x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/constants.ts

```diff
@@ -547,3 +547,160 @@ export const SOURCE_OVERVIEW_TITLE = i18n.translate(
     defaultMessage: 'Source overview',
   }
 );
+
+export const SOURCE_SYNCRONIZATION_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sources.sourceSyncronizationDescription',
+  {
+    defaultMessage:
+      'Sync chupa chups dragée gummi bears jelly beans brownie. Fruitcake pie chocolate cake caramels carrot cake cotton candy dragée sweet roll soufflé.',
+  }
+);
+
+export const SOURCE_FREQUENCY_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sources.sourceFrequencyDescription',
+  {
+    defaultMessage:
+      'Frequency chupa chups dragée gummi bears jelly beans brownie. Fruitcake pie chocolate cake caramels carrot cake cotton candy dragée sweet roll soufflé.',
+  }
+);
+
+export const SOURCE_OBJECTS_AND_ASSETS_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sources.sourceObjectsAndAssetsDescription',
+  {
+    defaultMessage:
+      'Objects chupa chups dragée gummi bears jelly beans brownie. Fruitcake pie chocolate cake caramels carrot cake cotton candy dragée sweet roll soufflé.',
+  }
+);
```

Comment:
To avoid translators being confused about these messages, maybe we could add something like this at the beginning of the text?

"DO NOT TRANSLATE, temporary placeholder:"